tests/test_serializer_fields.py

```diff
@@ -0,0 +1,75 @@
+from dateutil.relativedelta import relativedelta
+from rest_framework import serializers, viewsets
+
+from vng_api_common.serializers import DurationField
+
+
+class DurationSerializer(serializers.Serializer):
+    duration = DurationField()
+
+
+class DurationView(viewsets.ModelViewSet):
+    serializer_class = DurationSerializer
+
+
+def test_duration_field_internal_data():
+    data = {"duration": "P10D"}
+    serializer = DurationSerializer(data=data)
+    assert serializer.is_valid()
+    assert serializer.to_internal_value(data) == {"duration": relativedelta(days=+10)}
+
+
+def test_duration_to_representation():
```

Comment:
Could you also add multiple `duration_options` similar to the test for deserialization?
```suggestion
def test_duration_field_serialization():
```